src/common/popup/components/error-details.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import StackTrace from 'stacktrace-js';
+
+import CopyButton from './copy-button';
+
+import ErrorShape from '../utils/error-shape';
+
+class ErrorDetails extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      stackframes: null,
+    };
+  }
+
+  componentDidMount() {
+    const { error } = this.props;
+
+    StackTrace.fromError(error)
+      .then(stackframes => this.setState({ stackframes }))
+      .catch(err => console.error(err));
+  }
+
+  render() {
+    const { error } = this.props;
+    const { stackframes } = this.state;
+
+    if (!stackframes) return null;
+
+    const stack = stackframes.map(sf => sf.toString()).join('\n');
+    const info = [error.message, stack].join('\n\n');
+
+    return (
+      <div className="p-2 text-left">
+        <p className="font-weight-bold">{error.message}</p>
+        <pre className="overflow-auto">
+          <code>{stack}</code>
+        </pre>
```

Comment:
does it maybe make sense to use a readonly textarea instead, so it is scrollable by default? 
At least this approach seems quite common :)

Reply:
Not sure what benefit a textarea would give us here, esp. because we'd then need to style the textarea to use a monospaced code font etc.

Reply:
just thought it may be visually nicer, since the text is wierdly cut and it may not be obvious for a user that this is scrollable. But on the other hand 🤷‍♂  not to important :D